_posts/blog/2019-06-20-Hello!I am Yini He.md

```diff
@@ -0,0 +1,76 @@
+
+I love travelling around the world. The coexistence of the classics with the prosperity of London attracts me incredibly.  The people there are gentle and full of artistic temperament.  Modern and beautiful Vancouver attracts me just as much.  The people there understand the beauty of cultural integration.  Da Nang beckons to me with it’s mysteriousness and simplicity. The people there are warm and kind. 
+
+<figure>
+  <img src="/images/YiniHe/2panda4.jpg">
+</figure>
+                  
+At the same time, I am interested in all kinds of traditional Chinese culture, with its food, traditional clothing, calligaraphy, tea and Guqin(an ancient instrument of China). With a history of 5000 years, China has preserved its ancient philosophy and beautiful civilization.  More often, I am fascinated by the merging of Chinese and western cultures. I used to play the ukelele wearing Qipao(a traditional dress of China).  I also tried cooking a Chinese meal with fish and chips. 
+
+<figure>
+  <img src="/images/YiniHe/3qipao3.jpg">
+</figure>
+
+# Life and work goals
+
+--Ideal comes from life（理想源于生活）
+
+My university major is psychology.  In my senior year, I decided to go to a special school and work as an intern teacher there.  The students at this school were suffering from mental disorder. This was the first time that I truly understood the importance of mental health at heart.  I love those children even when some of them say things that are illogical and incoherent, while still others just always sat in a corner. But there are those who have great artistic talents.  Some of them are attractive and have a friendly disposition, willing to be best friends with me. Ever since then, I’ve always thought about these teenagers and hope that they could have a better life with the help of medical and therapeutic advancements for mental health. 
+
+<figure>
+  <img src="/images/YiniHe/4yuanmingyuan3.jpg">
+</figure>
+
+
+# My research experience
+
+Look at the fun of work, so we have fun every day(快乐生活，快乐工作)
+
+Ever since my experience as an intern at the school for special children, I have been concerned about the susceptibility to mental illness.  My major was modified to Neuroimaging.  The brilliance of how the brain works totally amazed me. I became very interested in regions of the brain, the connectivity and the fiber bundle of the brain where the source for the link on human behavior and emotion could be found and studied.
+
+I really appreciate that I have a great supervisor who is Tianzi Jiang.  He is always rigorous and enthusiastic about Neuroscience. I have to say from the heart, that he is one of the best role model of mine.  
+Now I came to the Turing Institute and I was pleasantly surprised here. I love Kirstie’s energy and she always give supports to me. She told me that I should do what I am most interested in. I hope to learn from her how to do the most convincing research. I also hope to learn from her passion to try to change the world.
```

Comment:
New paragraph?

Reply:
Yes.It is a new paragraph.Thank you very much for your careful revision.I've learned a lot from your advice